Umbraco-Heartcore/API-Documentation/Redirect/index.md

```diff
@@ -6,11 +6,14 @@ meta.Description: "Documentation for Heartcore Redirect APIs"
 
 # Redirect API
 
-This is the read-only API for delivering redirects, caused by moving or renaming content in the umbraco backoffice, to any app, website, device or platform.
+This is the read-only API for delivering redirects, caused by moving or renaming content in the umbraco backoffice, to any app, website, device, or platform.
+
+### NOTE
```

Comment:
Can you add the note like this:

:::note
The redirect API is only available if your project is using the **Content Delivery Platform**. You can verify this in the umbraco backoffice, settings section in the headless overview panel.
:::

Then it should look like a nice note in the documentation :D

Reply:
Thanks that is done now - i was not familiar with this syntax. My markup previewer also doesn't show it nicely - is it umbracoDocs specific, did i make a typo, or should i maybe get a new previewer ? :P